modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverrideExportImport.testcase

```diff
@@ -21,6 +21,40 @@ definition {
 		}
 	}
 
+	@description = "This test asserts that Cancel button works as expected in Import Translations page."
+	@priority = "5"
+	test CancelButtonWorks {
```

Comment:
Shall we rename the test to CanCancelImportTranslation. Thanks

Reply:
Done.